### ISSUE TITLE
508-defect-4 [IMPROVEMENT]: STEM Standalone - CONSIDER using a higher resolution image for the personal check in Step 5

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12343]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The personal check image is a bit blurry on higher resolution screens. It might be worth considering using a higher resolution image to illustrate the points about routing numbers and account numbers. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Environment
* https://staging.va.gov/education/apply-for-education-benefits/application/10203/personal-information/direct-deposit

## Screenshots or Trace Logs

![12343 Check Image.png](https://images.zenhubusercontent.com/60e78c6b02152f0a04637267/5678d349-5d14-49e7-9846-a196de85c1f1)

**Comment by desireeturner** [comment]
@1Copenut As discussed, this issue appears to be form system-wide.

**Comment by 1Copenut** [comment]
@meganhkelley This was flagged in my staging review for BAH's STEM standalone form. @desireeturner let me know this appears to be a forms lib issue, so I've reassigned to you. LMK if I need to post it elsewhere.

**Comment by bkjohnson** [comment]
Here is where we're accessing the image:

https://github.com/department-of-veterans-affairs/vets-website/blob/ce16b2de8305278b76d88df53df5db0cf474c4ce/src/platform/forms-system/src/js/definitions/content/directDeposit.jsx#L10-L11

And here is the file itself: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/assets/img/direct-deposit-check-guide.png

You may be able to get a higher resolution `png` by creating an export from [the svg file](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/assets/img/direct-deposit-check-guide.svg). That svg doesn't look right to me though, so a better option may be to create an entirely new SVG and use that instead of the PNG image. I'm not sure how much I will be able to help with this.

cc @cvalarida @caw310

**Comment by jfinnern** [comment]
Validated in local environment

**Comment by dsrr47** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/19300

**Comment by ERRonat0609** [comment]
Need to work with a UX/UI person to get a better image created in PNG format.

**Comment by af-manish** [comment]
@seanasewell Thanks for helping with this.

**Comment by seanasewell** [comment]
@af-manish Here ya go! Let me know if this doesn't work for you. Size is 2000px by 1006px.

![direct-deposit-feature.png](https://images.zenhubusercontent.com/615f0ad134e89a0c664dc300/1ca62557-623e-49c1-9df7-42640848a369)

**Comment by af-manish** [comment]
@seanasewell  Thanks. One requirement they have, sorry I should have mentioned, is that they want the routing number and account number to be all 0s.  Can you update with that?

**Comment by seanasewell** [comment]
@af-manish Not a problem! I am assuming the check number area should still be blank?

<img width="1000" alt="direct-deposit-check-guide" src="https://user-images.githubusercontent.com/83789847/146074259-0bc4d651-f1fd-485b-a58f-2d4d3b30aa06.png">

**Comment by af-manish** [comment]
@seanasewell  One more question - I found out that actually they want a SVG to be able to easy to maintain. I thought they landed at png. <it's been a bit confusing>. Is it simple to provide a SVG?

**Comment by seanasewell** [comment]
@af-manish I can provide that however I could not upload SVG as a comment. I am going to slack you the file. Let me know if you get it. Thanks!

**Comment by af-manish** [comment]
[direct-deposit-check-guide.svg.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/7720449/direct-deposit-check-guide.svg.zip)

I'm attaching the svg as a zip file here.

**Comment by jfinnern** [comment]
Validated in staging
Test run: https://dsvavsp.testrail.io/index.php?/runs/view/2768&group_by=cases:title&group_order=asc
Test case: https://afs365.sharepoint.com/:x:/r/sites/DGIBillServicesProject/Shared%20Documents/Legacy%20AO/Team%203/QA/Test%20Results/AFS-Sprint%205/508-defect-4%20%5BIMPROVEMENT%5D%20-%20STEM%20Standalone%20-%20CONSIDER%20using%20a%20higher%20resolution%20image%20for%20the%20personal%20check%20in%20Step%205.xlsx?d=w4afe1ea46289406785c0076b97fd044f&csf=1&web=1&e=dIFIal

**Comment by mrblanco1** [comment]
EDU UAT occurred and passed on 12/30/2021
